### DISCUSSION TITLE
Shortcuts to effects (?)

### DISCUSSION THREAD

**Discussion by F-Burning** [opentoonz/opentoonz#6557] [Ideas]
I think include effects in shortcut's list will improve the experience in the schematic panel 👍

**Comment by gab3d** [comment]
mmm... seems too much to me.
what other software do to this respect?

**Comment by F-Burning** [comment]
> mmm... seems too much to me.
> what other software do to this respect?

To me too 🤣 

But for example in Natron you can

https://user-images.githubusercontent.com/74098734/116584966-d0f12400-a90f-11eb-913b-0f504a517dac.mp4

I did press T to transform node, G for grade, and M for Merge. Then failed at the end 👓 

In Blender you can press Ctrl + T to create a transform nodes too.

Maybe make this shortcuts only works when you are in the Schematic?

**Comment by gab3d** [comment]
One good related feature Maya has in its Node Editor is the posibility of hitting the Tab key to have a node search field shown right at the mouse cursor position, so while you type a string the program shows in real-time the matching nodes in a list. When you see the one you're looking for appear in the list, you can use the Arrow keys to select it, and hit Enter to add that node to the editor.
It's not the same, but is a good way to avoid having to go and ooen another window to search for what you want.

I don't know how such a system would allow for Adding / Inserting / Replacing a node, though.
Perhaps depending on what is selected at the time of hitting the Tab key, or by using Shift / Ctrl + Tab for those other alternatives...?

**Comment by F-Burning** [comment]
That's similar to the free Fx console plugin of After Effects, a great alternative too, I like it.

> Perhaps depending on what is selected at the time of hitting the Tab key, or by using Shift / Ctrl + Tab for those other alternatives...?

Yes, Shift, Ctrl and Alt could work I think.
